Algorithms/Ruby/difference.rb

```diff
@@ -0,0 +1,38 @@
+# Write a function (similar to _.difference) that takes two arrays and returns two arrays containing the elements not present in the other
+def difference(arr1, arr2)
+  result_1 = arr1.reject do |num|
```

Comment:
One liners should use the simplified block syntax (as long as they aren't longer than the page width)